charts/atlas-operator/templates/cluster-roles.yaml

```diff
@@ -0,0 +1,65 @@
+{{- $operatorName := include "mongodb-atlas-operator.name" . -}}
+
+{{- if not .Values.watchNamespaces }}
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: "{{ $operatorName }}"
+  labels:
+  {{- include "mongodb-atlas-operator.labels" $ | nindent 4 }}
+rules:
+{{ template "mongodb-atlas-operator.rbacRules" $ | toYaml | indent 2 }}
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ $operatorName }}
+  labels:
+  {{- include "mongodb-atlas-operator.labels" $ | nindent 4 }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ $operatorName }}
+subjects:
+  - kind: ServiceAccount
+    name: {{ include "mongodb-atlas-operator.serviceAccountName" . }}
+    namespace: {{ $.Release.Namespace }}
+
+{{- end }}
+
+{{- /* It seems that the rbac-proxy role must be Cluster-scoped to handle requests from different namespaces?
+TODO May make sense in the future to configure a specific namespace where the prometheus server resides to give Role only to that namespace? */}}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
the role necessary for the rbac-proxy to work correctly. So far Clusterwide but should be limited to the namespace where the prometheus servers is installed